### PR TITLE
Refactored Hooks API

### PR DESCRIPTION
**Pull request by nicosomb**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Refactored Hooks API: delete hook-status endpoint and change /hooks/id to /hook/id
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| Sponsor company   | PrestaShop SA
| How to test?      | CI is green

### CODE REVIEW

**Review by jolelievre — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
